### PR TITLE
chore: run image as node user

### PR DESCRIPTION
**Pull request by jbdelpech**
Run elastalert image as unpriviledged user, kubernetes compatible.
BTW, add a default empty dir in /opt/elastalert/rules.

If you have custom mounted volumes : it represents a BREAKING CHANGE for all images currently running as root in your cluster.

Before upgrading to this image : change your directory rights to 1000:1000 (or node:node into old elastalert image).

Feel free to test it, I packaged it from bitsensor/elastalert:2.0.1 with the changes of this PR : `solocal/elastalert:2.0.3`

Do not use it in production, this image will be removed soon.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -45,5 +45,12 @@ COPY config/config.json config/config.json
 COPY rule_templates/ /opt/elastalert/rule_templates
 COPY elastalert_modules/ /opt/elastalert/elastalert_modules
 
+# Add default rules directory
+# Set permission as unpriviledged user (1000:1000), compatible with Kubernetes
+RUN mkdir -p /opt/elastalert/rules/ \
```
Comment (jBouyoud) [inline Dockerfile]: Test folder seems to be missing : `/opt/elastalert/server_data/tests/`

Comment (jbdelpech) [reply Dockerfile]: Thanks for your feedback !

I fixed it !

@martijnrondeel Are you interested by this change ?

**Comment by martijnrondeel** [comment]
Thank you for this merge request @jbdelpech :+1:, it will be in the next beta release.

**Comment by martijnrondeel** [comment]
This is available now in [3.0.0-beta.0](https://github.com/bitsensor/elastalert/releases/tag/3.0.0-beta.0).

**Pull request merged**